neqo-transport/src/connection/mod.rs

```diff
@@ -424,13 +492,13 @@ impl Connection {
             recv_streams: RecvStreams::default(),
             flow_mgr: Rc::new(RefCell::new(FlowMgr::default())),
             state_signaling: StateSignaling::Idle,
-            loss_recovery: LossRecovery::new(cc_algorithm, stats.clone()),
+            loss_recovery: LossRecovery::new(conn_params.cc_algorithm, stats.clone()),
             events: ConnectionEvents::default(),
             new_token: NewTokenState::new(role),
             stats,
             qlog: NeqoQlog::disabled(),
             release_resumption_token_timer: None,
-            quic_version,
+            quic_version: conn_params.quic_version,
```

Comment:
Would it make sense to keep a copy of the parameters instead?  They are immutable, so there is no risk of them become inconsistent with anything that used them.

Reply:
We only need one member of ConnectionParameters  (soon we will need cc_algorithm as well) to be remembered in Connection. Therefore let's not hold the complete ConnectionParameters.